### PR TITLE
Additional testing

### PR DESCRIPTION
**Pull request by elisarver**
During my additional testing, I found that Stop was more of a Kill, and re-added a pass-through on Wait() that captures the exit code to the command before returning.

I also increased coverage a bit. Most stuff beside the err != nil cases are covered.

### CODE REVIEW

**Comment by elisarver** [comment]
I've significantly re-worked this and it's a *lot* simpler than it was before. I'm basically doing the wrapper technique with `*exec.Cmd` and it's a lot more straightforward for those previously having used that functionality.

File: README.md [hunk 1]
```diff
@@ -1,48 +1,100 @@
 # command
 
-This package wraps the functionality of calling an `*exec.Cmd.CombinedOutput()`, capturing an easy-to-serialize result
-for either re-running, or writing out to disk.
+This package wraps the functionality of `*exec.Cmd` with some simple IO tools to make creating a `Cmd` easier. It provides
+the
```
Comment (shairozan) [inline README.md]: Did we need to finish the thought or what does it provide?

**Review by shairozan — APPROVED** [review]
Looks good. Had to dig into Cmd to find the Process object and Kill since it was embedded. I had one comment about the readme which I wasn't sure if it was an unfinished thought or not

**Comment by elisarver** [comment]
I'll need to mainline this branch for demo, so I'm going to merge the PR.

**Pull request merged**